tests/test_misc.py

```diff
@@ -407,7 +407,27 @@ def fn(a):
             self, fn=fn, nargs=1, expected_ops=5, expected_ops_dynamic=8
         )
 
-    def test_tensor_item(self):
+    def test_tensor_item_capture(self):
```

Comment:
```suggestion
    @patch.object(torchdynamo.config, "capture_scalar_outputs", True)
    def test_tensor_item_capture(self):
```

Might make the tests more readable.